tests_tensorflow/resources/buyer/index.html

```diff
@@ -0,0 +1,28 @@
+<script>
+function randomVector(n) {
+  const vector = new Array(n);
+  for (let i = 0; i < n; i++) {
+    vector[i] = Math.random();
+  }
+  return vector;
+}
+
+if (!navigator.joinAdInterestGroup) {
+  document.write("your browser doesn't have joinAdInterestGroup -- do you need to set some flags?");
+} else {
+  navigator.joinAdInterestGroup({
+    owner: window.location.origin,
+    name: "tc-ig",
+    biddingLogicUrl: window.location.origin + "/buyer_browser.js",
```

Comment:
+1 for relative path, it would be more readable by defining:

const buyer_address = window.location.origin;